geometry/distance.c

```diff
@@ -0,0 +1,37 @@
+#include <math.h>
+#include <stdio.h>
+/**
+ * Author Arnab Nath
+ * The code is implemented in VS Code
+ */
+/*
+ * imported the math.h header to implement various mathematical functions.
+ * imported the stlib.h header whidh is the general standard  purpose library of
+ * C programming language.
+ */
+
+// Function to calculate distance between 2 points
+// The X and Y coordinates are the parameters of the function
+float distance(int x1, int y1, int x2, int y2)
```

Comment:
This should include parameter documentation, what does the function return, etc..